mailpoet/assets/js/src/form_editor/components/sidebar/default_sidebar.tsx

```diff
@@ -8,6 +12,14 @@ import { BlockSettings } from './block_settings.jsx';
 import { SidebarHeader } from './sidebar_header';
 import { store } from '../../store';
 
+// workaround for block editor store useSelect
```

Comment:
Was this later removed (moved to `types/...` as per https://github.com/mailpoet/mailpoet/pull/4786/commits/5638e314cd633498efe6a247dcc6328775d77175)? Need to check.